examples/Dockerfile

```diff
@@ -1,4 +1,3 @@
-#FROM python:3.8.13-bullseye@sha256:2a01d88a1684e6d7f08030cf5ae73b536926c64076cab197e9e3d9f699255283
 FROM ubuntu:20.04
```

Comment:
Why is this changed? It's much safer to pin to a particular commit of an image to avoid failures showing up with no changes on our end.